views/status.jade

```diff
@@ -0,0 +1,115 @@
+extends ./layout.jade
+
+block scripts
+    <!-- CSS includes -->
```

Comment:
same as rest of pages, however, this could definitely be something to change in the future for all the files